packages/commons-server/src/libs/server/server.ts

```diff
@@ -531,6 +531,13 @@ export class MockoonServer extends (EventEmitter as new () => TypedEmitter<Serve
         );
       }
 
+      if (request.locals) {
+        const statusCode = request.locals.get('statusCode');
+        if (statusCode) {
+          response.status(statusCode);
+        }
+      }
+
```

Comment:
This should be replicated in the `sendFile` method too (line 627-628), as the templating system is supported in files.

Reply:
Ok, no prob, I'm going to add it on sendFile too